src/isomorphic-fetch.js

```diff
@@ -1,9 +1,16 @@
+/* globals require */
+
 import global from './metal/global';
 import isNodeLikeEnvironment from './metal/is-node-like-environment';
 
 const fetch = global.fetch;
 
 if (!fetch && isNodeLikeEnvironment()) {
-  global.fetch = global.require('node-fetch');
+  /* global.require is only available in the REPL 
+   * and Babel fails when you use require() explicitly
+   */
```

Comment:
@minasmart, i get what you mean now. the problem also happens when transpiling to `umd`. It's a problem with babel6. I found situations when it choked too when transpiling scripts with `decorators`. I have updated the comment now